docs/source/contributing/releasing.adoc

```diff
@@ -94,6 +106,13 @@ For more information, see xref:../contributing/writing-docs.adoc#[Writing and Pu
 [[post-release-tasks]]
 === Post-Release Tasks
 
+The systemtray bits also needs to be uploaded to the release in Github.
+Create {project} binaries with systemtray and upload them to Github (works only in macOS):
```

Comment:
@anjannath You should mention here that `make release_systemtray` needs to be done on a macOS system. Also, mention about the prerequisites before running the command.

Reply:
@LalatenduMohanty Prerequisites are mentioned in its own section, along with the others.. also in the bracket it says (works only in macOS).